### ISSUE TITLE
[🐛 BUG] Weird syntax when editing a boolean in Table

### ISSUE THREAD

**Issue by FlorianJacta** [Avaiga/taipy#2257]
### What went wrong? 🤔

Editing a cell representing a boolean will create a weird syntax shown on the UI while editing.

![image](https://github.com/user-attachments/assets/bcff03c6-f80f-49e8-bd72-d3618a2fc0e0)

### Expected Behavior

We should not see this syntax.

### Steps to Reproduce Issue

Run this code and try to edit:

```python
import taipy.gui.builder as tgb
import pandas as pd
from taipy import Gui

data = pd.DataFrame(
    {"Include": [True, False, True], "A": [1, 2, 3], "B": [4, 5, 6], "C": [7, 8, 9]}
)

with tgb.Page() as page:
    tgb.table("{data}", editable__Include=True)

    tgb.table("{data}", editable__Include=True, use_checkbox=True)

Gui(page).run()

```

### Version of Taipy

4.0.*

### Acceptance Criteria

- [ ] A unit test reproducing the bug is added.
- [ ] Any new code is covered by a unit tested.
- [ ] Check code coverage is at least 90%.
- [ ] The bug reporter validated the fix.
- [ ] Related issue(s) in taipy-doc are created for documentation and Release Notes are updated.

### Code of Conduct

- [X] I have checked the [existing issues](https://github.com/Avaiga/taipy/issues?q=is%3Aissue+).
- [ ] I am willing to work on this issue (optional)

**Comment by FredLL-Avaiga** [comment]
CSS Workaround would be:
```py
style={
        ".taipy-table .tpc-editing": {
            ".MuiBox-root:has(.taipy-table-bool)": {"font-size": 0},
            "input.taipy-table-bool": {"display": "none"},
        }
```

**Comment by FlorianJacta** [comment]
Fixed in 4.0.2
